osmaxx/profile/views.py

```diff
@@ -18,14 +18,15 @@
 class SendVerificationEmailMixin(object):
     RATE_LIMIT_SECONDS = 30
 
-    def _send_email_verification(self, profile):
-        if cache.get(profile.associated_user.id):
+    def _send_email_verification(self):
```

Comment:
The current callers ensure this or at least assume it, otherwise they wouldn't have been able to pass the profile. But we can't really guarantee it for future callers or future revisions of the current callers.

Would this be reason to keep the profile argument?

An alternative refactoring would be to make `_send_email_verification` a public method of `models.Profile`.